### PR TITLE
Remove admin conditional in upgrade action

### PR DESCRIPTION
**Pull request by pipo02mix**
From now on we let all users upgrade clusters

### CODE REVIEW

File: src/components/cluster/detail/view.js [hunk 1]
```diff
@@ -147,21 +143,6 @@ class ClusterDetailView extends React.Component {
       return false;
     }
 
-    // cluster release_version must be >= 3 on AWS, >= 1 on Azure, >= 2.7.0 on KVM
```
Comment (pipo02mix) [inline src/components/cluster/detail/view.js]: Cleaning...

File: src/components/cluster/detail/view.js [hunk 2]
```diff
@@ -147,21 +142,6 @@ class ClusterDetailView extends React.Component {
...
-        return false;
-      }
-    } else if (this.props.provider === 'kvm') {
-      if (cmp(this.props.cluster.release_version, '2.7.0') === -1) {
-        return false;
-      }
-    }
-
     return true;
   }
```
[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/components/cluster/detail/view.js]: but you can be in the latest version, right?

[the remark this answers is not in the document]
Comment (marians) [reply src/components/cluster/detail/view.js]: Yes. But to my understanding, that is not what this function checks.

[the remark this answers is not in the document]
Comment (pipo02mix) [reply src/components/cluster/detail/view.js]: ok so ideally we need to rename the function to `isNewerReleaseAvailable`?

**Pull request merged**